forc-gm/Cargo.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "forc-gm"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+homepage = "https://fuel.network/"
+license = "Apache-2.0"
+repository = "https://github.com/FuelLabs/sway"
+description = "A sample `forc` plugin."
+
+[dependencies]
+clap = { version = "=3.1.8", features = ["derive"] }
```

major.minor unless there's a specific reason not to.
```suggestion
clap = { version = "3.1", features = ["derive"] }
```